src/main/java/core/basesyntax/service/FileWriteImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.FileWrite;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriteImpl implements FileWrite {
+    @Override
+    public void write(String result, String nameFileWrite) {
+        try {
+            Files.write(Paths.get(nameFileWrite), result.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Cant write file " + nameFileWrite, e);
+        }
+
+    }
```

Comment:
According to the checklist, it is recommended to use try-with-resources when dealing with I/O operations to ensure that all resources are properly closed. In this case, you are using Files.write() which handles resource closing for you, so this comment is just a reminder for cases when you manually open resources.